docker/pypi/dmwm-base/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM registry.cern.ch/cmsweb/cmsweb-base as cmsweb-base
-FROM registry.cern.ch/cmsweb/exporters as exporters
-FROM python:3.8-bullseye
+FROM registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506-stable as cmsweb-base
+FROM registry.cern.ch/cmsweb/exporters:20250519-stable as exporters
+FROM python:3.12-bookworm
```

Comment:
Dario, I wonder why did you choose `3.12-bookworm` image instead of smaller `python:3.12-slim` which is stripped-down version of the standard image, removing unnecessary files and tools to reduce size? Since we install all our tools via pip I don't see any more point in having full image and we most likely can use stripped one. Sorry for pointing this later, but it would be useful I think to test and reduce image size.

Reply:
done